src/config.rs

```diff
@@ -63,6 +63,7 @@ pub struct WindowConfig {
    pub y: i32,
    pub width: u32,
    pub height: u32,
+   pub maximized: bool,
```

Comment:
For backwards-compatibility purposes, please add `#[serde(default)]` to this field, so that parsing files without it will still succeed (and default to windowed mode.)